packages/cli/src/h2.cts

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env node
+
+const path = require('path');
+const {spawnSync} = require('child_process');
```

Comment:
What about simplifying this a bit by:
- Getting the arguments after `h2`
- Invoking the CLI with `npm exec -- shopify {args}`

`npm exec` should do the look up for you. That'll be safer than having logic that assumes the internal executable will be under `bin/run.js` because we are building an implicit contract.